packages/shorebird_cli/lib/src/commands/release/release_command.dart

```diff
@@ -358,19 +358,29 @@ Use `shorebird flutter versions list` to list available versions.
       // All artifacts associated with a given release must be built
       // with the same Flutter revision.
       if (existingRelease.flutterRevision != flutterRevision) {
+        final existingReleaseStringVersionString =
+            await shorebirdFlutter.getVersionAndRevision(
+          flutterRevision: existingRelease.flutterRevision,
+        );
+
+        final releaseVersionString =
+            await shorebirdFlutter.getVersionAndRevision(
+          flutterRevision: flutterRevision,
+        );
```

Comment:
Can't we revert `getVersionAndRevision` to not accept an optional `flutterRevision` and instead in this case get the Flutter version for the revision and output the formatted string?